src/bucket.ts

```diff
@@ -362,8 +362,6 @@ export interface UploadOptions
   destination?: string | File;
   encryptionKey?: string | Buffer;
   kmsKeyName?: string;
-  resumable?: boolean;
-  timeout?: number;
```

Comment:
Was this accidentally removed? It is still documented below.

Reply:
not accidentally removed and still documented below because it's still part of the interface. it's declared on `CreateResumableUploadOptions`. I removed this because it's duplicated